src/client/App.tsx

```diff
@@ -17,20 +17,15 @@ export const row = {
   tableLayout: 'fixed' as 'fixed'
 }
 
-class App extends Component {
-  componentDidMount(): void {
+export default function App(): JSX.Element {
+  useEffect(() => {
```

Comment:
I believe useEffect will run after every render if isn't passed a dependency array for its second argument. Look into passing an empty array as the second argument so that this effect is only ran after the first render: https://reactjs.org/docs/hooks-effect.html

Reply:
Good catch